content/theme/templates/menu.html

```diff
@@ -1,8 +1,10 @@
 <!-- nav bar -->
-<nav class="navbar navbar-expand-lg navbar-light bg-info" aria-label="Fifth navbar example">
+<nav class="navbar navbar-expand-lg navbar-dark bg-black" aria-label="Fifth navbar example">
```

Comment:
The new ASF logo looks much better with a light background. Please revert this change.